src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.ts

```diff
@@ -246,13 +284,13 @@ export class MongoDBEventStore implements EventStore<MongoDBReadEventMetadata> {
         readModels: stream?.projections ?? {},
         events: eventsToAppend,
         projections: this.inlineProjections,
-        collection: this.collection,
+        collection,
         updates,
         client: {},
```

Comment:
I didn't see this on the updates to projections, but for inline projections a client isn't necessary right? Or is it just because the context for inline projections and async projections (once implemented) will be the same type?

Reply:
Yes, that's true: for now. If we have a strategy to have a collection per stream type, then technically, this could be placed in a separate Mongo database. For that, we'd need the client to resolve the collection. Having collection in different databases would help to have e.g. separation of data storage per module or per tenant and still use a single event store singleton setup.